### ISSUE TITLE
[Bug] OpenBB Terminal do NOT accept API from FRED

### ISSUE THREAD

**Issue by aybarsarikaya** [OpenBB-finance/OpenBBTerminal#2182]
**Describe the bug**
However i've put the API on OpenBB Terminal. When i wanted to reach data, terminal giving me an error message.

**To Reproduce**
First of all i've put the API then i wrote the code like that:

2022 Jul 26, 06:19 (🦋) /economy/ $ fred OVXCLS -s 2008-01-01 -e 2015-01-01

Error: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)>

**Screenshots**
<img width="954" alt="Screen Shot 2022-07-26 at 13 34 22" src="https://user-images.githubusercontent.com/92736627/180986284-45f6ba09-ce82-4d51-8511-6502c6610e81.png">

By the way, i've tried Install Certificates option on Python, also, OpenBB Terminal do not have this option.

- OS: [e.g. macOS, M1]
- Python version [e.g. 3.10]

**Comment by JerBouma** [comment]
Does this occur to you in the installer version of the OpenBB Terminal (from [here](https://openbb.co/products/terminal))? It seems to work just fine on the Python version at least.

<img width="1380" alt="image" src="https://user-images.githubusercontent.com/46355364/180997851-82fd1d38-26e7-446f-ac17-77affab49579.png">

**Comment by aybarsarikaya** [comment]
Yes i've downloaded from there but i am still living that problem.

**Comment by JerBouma** [comment]
@piiq Could you look into this (or delegate)? This seems to be specific to the installer.

**Comment by deeleeramone** [comment]
Mac M1 installer:
![Screen Shot 2022-07-26 at 1 59 03 PM](https://user-images.githubusercontent.com/85772166/181110819-ae2333be-5048-46be-aaec-ff1b64144331.png)

**Comment by cheeeeee** [comment]
I was getting this error as well on the latest version. Updating anaconda with `conda update conda` and doing another poetry install seemed to fix things for me.

**Comment by simmonsj330** [comment]
@aybarsarikaya Updating to v1.8.0 seems to fix this issue on both Intel Mac and M1 Mac! Closing this issue!

<img width="822" alt="Screen Shot 2022-09-15 at 4 30 27 PM" src="https://user-images.githubusercontent.com/53658028/190502867-cca695e3-b4c5-44a0-8b4a-91edd0b59d61.png">
<img width="1338" alt="Screen Shot 2022-09-15 at 4 33 54 PM" src="https://user-images.githubusercontent.com/53658028/190503357-92c438f6-a7f5-491a-8c77-905fbf87c882.png">

**Comment by JerBouma** [comment]
Are you sure about that @simmonsj330?

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/46355364/190850930-97bd5925-5b74-4b5c-bdfa-fcf4c8016aad.png">

**Comment by catalintoma** [comment]
@JerBouma I can move this to another Bug if it is not related at all, but I've just installed the terminal yesterday and got the same error, but on the **stocks** menu

I know I am behind a proxy, so I did try to open that link in the browser and add those certificates to **certifi** but it made no difference.

The interesting thing is that I've looked at how [this repository](https://github.com/JerBouma/FinanceDatabase/blob/master/) does the call and I was able to make the call in a test script.

I guess what I am missing is some context as to what certificates the installer uses and how to extend them.

![openbb](https://user-images.githubusercontent.com/16468607/190893474-d0a71cd8-f5d9-489c-a78f-3c1afc8dd32a.png)
